### PR TITLE
krystophv/emoji

### PR DESCRIPTION
**Pull request by krystophv**
Replaces the unicode emoji with the standard colon-delimited style emoji present in Slack and Github. This is done through a Gitbook plugin which needs to be installed through the Gitbook CLI before the CLI will serve or build the docs, so I've added an NPM postinstall hook to take care of the Gitbook plugin install process at the same time. The Markdown files look a little strange on Github in the raw form because Github's emoji parser is a little greedier than the emoji plugin for Gitbook and will interpret the emoji strings even with no whitespace separation where Gitbook's plugin requires some whitespace around them to actually convert it.

### CODE REVIEW

**Comment by jebeck** [comment]
Hmm, the postinstall hook doesn't seem to be working for me. (Well, it appears to work, but then gitbook thinks the plugin isn't installed...) Thoughts, @krystophv?

``` bash
herodotus:docs jebeck$ npm install

> docs@0.0.1 postinstall /Volumes/Tidepool/docs
> gitbook install

info: 1 plugins to install
info: No version specified, resolve plugin advanced-emoji
info: install plugin advanced-emoji from npm (gitbook-plugin-advanced-emoji) with version 0.0.1
gitbook-plugin-advanced-emoji@0.0.1 node_modules/gitbook-plugin-advanced-emoji
info: >> plugin advanced-emoji installed with success

Done, without error
herodotus:docs jebeck$ npm run serve-docs

> docs@0.0.1 serve-docs /Volumes/Tidepool/docs
> gitbook serve

Live reload server started on port: 35729
Press CTRL+C to quit ...

info: loading book configuration....OK
info: load plugin advanced-emoji ....ERROR
info: load plugin gitbook-plugin-highlight ....OK
info: load plugin gitbook-plugin-livereload ....OK

Error: Error loading plugins: advanced-emoji. Run "gitbook install" to install plugins from NPM.
```

**Comment by jebeck** [comment]
Ah, a little hunting shows that there may be a GitBook issue here, or an issue with the plugin itself: https://github.com/GitbookIO/gitbook/issues/1151#issue-135031965 (see [my comment](https://github.com/GitbookIO/gitbook/issues/1151#issuecomment-190475950) at the bottom)

@krystophv I gather you didn't try locally yourself? Actually, your comments about the whitespace & greediness make it sound as if you did. So hmm, wonder what's up here. Are you in node 0.12.0? Can you give steps that you can replicate for serving and/or building the docs successfully?

File: GitHub/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In all application and library repositories, we use the `master` branch for prod
...
+All significant changes should be made through a pull request that is approved by another Tidepool developer through a process of code review. We often use LGTM "looks good to me" as our approval sign-off at the conclusion of a review and/or one or more rounds of feedback and changes. One of the following (or any similarly unambiguous) emoji is also acceptable as sign-off: :ship: (:ship:), :thumbsup: (:thumbsup:), or the GitHub ["ship it squirrel"](https://www.quora.com/GitHub/What-is-the-significance-of-the-Ship-It-squirrel 'Quora: Ship It Squirrel') :shipit: (:shipit:,:squirrel:).
```
Comment (jebeck) [inline GitHub/README.md]: You'll want to use some kind of ignore or literal syntax where you actually want the `:ship:` colon-delimited shortcode to show up...

**Comment by krystophv** [comment]
@jebeck I'm not sure why it's not working for you. Here's what I did (killed `node_modules` just to be sure):

``` bash
krystophv at makashi in ~/tidepool/docs on krystophv/emoji
$ rm -rf node_modules

krystophv at makashi in ~/tidepool/docs on krystophv/emoji
$ node --version
v0.12.10

krystophv at makashi in ~/tidepool/docs on krystophv/emoji
$ npm install
-
> docs@0.0.1 postinstall /Users/krystophv/tidepool/docs
> gitbook install

info: 1 plugins to install
info: No version specified, resolve plugin advanced-emoji
info: install plugin advanced-emoji from npm (gitbook-plugin-advanced-emoji) with version 0.1.6
gitbook-plugin-advanced-emoji@0.1.6 node_modules/gitbook-plugin-advanced-emoji
└── emojify.js@1.1.0
info: >> plugin advanced-emoji installed with success

Done, without error
gitbook-cli@1.0.1 node_modules/gitbook-cli
├── bash-color@0.0.3
├── q@1.4.1
├── semver@5.0.1
├── npmi@1.0.1 (semver@4.3.6)
├── user-home@2.0.0 (os-homedir@1.0.1)
├── commander@2.8.1 (graceful-readlink@1.0.1)
├── tmp@0.0.27 (os-tmpdir@1.0.1)
├── optimist@0.6.1 (wordwrap@0.0.3, minimist@0.0.10)
├── lodash@3.10.1
├── fs-extra@0.24.0 (path-is-absolute@1.0.0, jsonfile@2.2.3, graceful-fs@4.1.3, rimraf@2.5.2)
└── npm@2.14.8

krystophv at makashi in ~/tidepool/docs on krystophv/emoji
$ npm run serve-docs

> docs@0.0.1 serve-docs /Users/krystophv/tidepool/docs
> gitbook serve

Live reload server started on port: 35729
Press CTRL+C to quit ...

info: loading book configuration....OK
info: load plugin gitbook-plugin-advanced-emoji ....OK
info: load plugin gitbook-plugin-highlight ....OK
info: load plugin gitbook-plugin-search ....OK
info: load plugin gitbook-plugin-sharing ....OK
info: load plugin gitbook-plugin-fontsettings ....OK
info: load plugin gitbook-plugin-livereload ....OK
info: >> 6 plugins loaded
info: start generation with website generator
info: clean website generator
info: OK
info: generation is finished

Starting server ...
Serving book on http://localhost:4000
```

It may be that the version of gitbook you have installed globally isn't new enough? Have you tried serving through the `npm run` command? I'm on node 0.12.10 (since the patch versions shouldn't have any breaking changes, I went for the latest 0.12)

**Comment by jebeck** [comment]
So I'm unable to get the emoji plugin working despite quite a lot of effort. Giving up for now, and we'll just let this PR sit (can't merge when it prevents me from serving or building, even if @krystophv can). 😕

**Comment by jebeck** [comment]
Now that I can build it, merging & publishing :ship:

**Pull request merged**